network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/prekey/PrekeyApiTest.kt

```diff
@@ -25,13 +25,13 @@ class PrekeyApiTest : ApiTest {
             assertion = {
                 assertPost()
                 assertJson()
-                assertPathEqual(PATH_PREYKEY)
+                assertPathEqual(PATH_PREKEYS)
             }
         )
         val preKeyApi: PreKeyApi = PreKeyApiImpl(httpClient)
 
         val response = preKeyApi.getUsersPreKey(VALID_GET_USERS_PREKEY_REQUEST.serializableData)
-        assertEquals(response.resultBody, VALID_GET_USERS_PREKEY_RESPONSE.serializableData)
+        assertEquals(response.value(), VALID_GET_USERS_PREKEY_RESPONSE.serializableData)
```

Comment:
That does not compile.

This does:
```suggestion
        assertTrue(response.isSuccessful())
        assertEquals(response.value, VALID_GET_USERS_PREKEY_RESPONSE.serializableData)
```